.github/workflows/release.yml

```diff
@@ -24,14 +24,15 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Import environment variables from file
+        run: cat ".github/env" >> "$GITHUB_ENV"
+
       - name: Setup Go
         uses: actions/setup-go@f111f3307d8850f501ac008e886eec1fd1932a34 # pin@5.3.0
         with:
           go-version: '${{ env.golang-version }}'
-
       - name: Run unit tests
         run: make test-coverage
-
```

Comment:
I wouldn't remove the blank lines around "Run unit tests" step, since the rest have blank lines let's keep it consistent.